docs/root/configuration/http/http_filters/_include/grpc-reverse-bridge-filter.yaml

```diff
@@ -1,5 +1,9 @@
 admin:
-  access_log_path: /dev/stdout
+  access_log:
+  - name: envoy.access_loggers.file
+    typed_config:
+      "@type": type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+      path: /dev/stdout
```

Comment:
This would change with your other PR right? Not sure which one we should land first. Probably the other one?

Reply:
Lets do the other one first. Seems to be in later stage.

Reply:
fixed after picking up main